test/getBlobDuration.test.js

```diff
@@ -52,3 +57,12 @@ it('should execute Chrome bugfix duration retrieval as needed', async () => {
     resolve()
   })
 })
+
+it('should reject with the error object if an error occurs', async () => {
+  dummyVideoEl.duration = Infinity
+
+  // noinspection ES6MissingAwait
+  const durationP = getBlobDuration(mockBlob)
+  dummyVideoEl.onerror(dummyErrorEventObject)
+  expect(durationP).rejects.toMatch(dummyErrorEventObject.target.error)
```

Comment:
I believe this should be `await`ed; the manual somewhat agrees: https://jestjs.io/docs/tutorial-async#rejects

Reply:
Nope, not intentional, that one I definitely missed, thanks for catching it!